release/models/network-instance/openconfig-network-instance-l2.yang

```diff
@@ -698,6 +709,12 @@ submodule openconfig-network-instance-l2 {
       reference "RFC7432: BGP MPLS-Based Ethernet VPN";
     }
 
+    leaf next-hop-group {
+      type leafref {
+        path "../../../../../../next-hop-groups/next-hop-group/id";
+      }
+      description "Leafref next-hop for the MAC-IP table entry";
```

Comment:
should it be next-hop-group instead of next-hop?

Reply:
This was the wrong change actually.  This leaf needs to remain unchanged but `deprecated` and the description updated for `next-hop-group` below.